packages/medusa/src/loaders/index.ts

```diff
@@ -164,6 +163,8 @@ export default async ({
     expressApp,
     rootDirectory
   )
+
+  const { createDefaultsWorkflow } = await import("@medusajs/core-flows")
```

Comment:
Maybe we need to revisit this decision of calling `createDefaultsWorkflow` when starting up the app, it is probably better suited to be run as part of `seed`. It is incomplete in what it creates anyway (eg. it doesn't create a shipping profile, which is required to create a shipping option).

cc @olivermrbl WDYT?

Reply:
Yeah, think it should be fine to remove this in favor of a defaults seed script. Lemme create a ticket for this

Reply:
yeah I dont think it should be part of this pr but yes agree 👍